tests/integration/test_manylinux.py

```diff
@@ -23,13 +24,15 @@
     'manylinux2010': MANYLINUX2010_IMAGE_ID,
 }
 DOCKER_CONTAINER_NAME = 'auditwheel-test-manylinux'
-PYTHON_IMAGE_ID = 'python:3.5'
+PYTHON_MAJ_MIN = [str(i) for i in sys.version_info[:2]]
+PYTHON_ABI = 'cp{0}-cp{0}m'.format(''.join(PYTHON_MAJ_MIN))
+PYTHON_IMAGE_ID = 'python:' + '.'.join(PYTHON_MAJ_MIN)
 DEVTOOLSET = {
     'manylinux1': 'devtoolset-2',
     'manylinux2010': 'devtoolset-8',
 }
 PATH_DIRS = [
-    '/opt/python/cp35-cp35m/bin',
+    '/opt/python/{}/bin'.format(PYTHON_ABI),
```

Comment:
If you want to make this dynamic, you could just do `os.path.dirname(sys.executable)`.

Reply:
We can't if I'm not mistaken. `sys.executable` relates to the `python` interpreter running the tests while `/opt/python/{}/bin'.format(PYTHON_ABI)` relates to the `python` interpreter running in the container.